src/site/_collections/post-to-collections.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = (collections) => {
+  const postToCollections = {};
+  const paths = collections
+    .getFilteredByTag("pathItem")
+    .map((p) => p.data.path);
```

Comment:
I think the array that this returns is larger than it needs to be. I think we have <20 paths, but this returns an array of a few hundred items which are just the same path object duplicated in many instances.

Reply:
I'm not sure what's the best way to address that though, I checked and it looks like i get around 260 items...

Reply:
Would something like this work?

```
const paths = new Map();
collections.getFilteredByTag("pathItem").forEach((p) => {
  const path = p.data.path;
  paths.set(path, path);
});

paths.forEach((path) => ...
```